docker/solr/solr-language-hack.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Would this be a better fit here?  https://github.com/open-oni/open-oni-scripts

Reply:
Hmm, probably